docs/2.develop/3.development/anatomy/anatomy.md

```diff
@@ -0,0 +1,138 @@
+
+  ```rust
+    use near_sdk::{env, near_bindgen, AccountId};
+    use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+  ```
+
+  </TabItem>
+</Tabs>
+
+The NEAR SDK defines methods to, among other things:
+
+1. Understand the context of the call (e.g. who started it, how much money they sent).
+2. Handle the state (storage) of the smart contract.
+3. Transfer money to other users/contracts.
+4. Interact with other smart contracts.
+
+---
+### Contract's Interface
+Smart contracts expose an interface so users in the blockchain can interact with them. A contract's interface is made of all the public methods that live in the main file. In our example, the public methods which anyone could call are:
+
+1. `init`: This method can only be called once. It defines the value for the `beneficiary` variable. 
+2. `donate`: A method in which the users attach NEARs in order to make a donation.
+3. `get_donation_by_number`: Returns a object with two fields: account_id, and amount. Represents a donation recorded in the system.
+
+All the other methods remain private, and can only be called from within the contract.
+
+An important difference between RUST and Assemblyscript is that in RUST the smartcontract is an object. Therefore, in RUST the contract's attributes are stored in `self`. In contrast, in Assemblyscript we need to explicitly rely on the `storage` object to store all the attributes.
+
+:::tip
+In RUST we are also relying in the `env::storage` object to store the contract's attributes. However, this gets abstracted away by the SDK.
+:::
+
+:::warning
+In Assemblyscript you could call multiple times the `init` function. Please make sure to add a check so the function is not called more than once. This can be done by simply adding the following code:
+
+```ts
+  const initialized: bool = storage.getPrimitive<bool>('initialized', false)
+  assert(!initialized, "Already initialized")
+  storage.set<bool>('initialized', true)
+```
+:::
+
+---
+
+### Classes and NEAR Bindgen
+
+You might have notice in the examples that the classes are decorated with `nearbindgen`:
+
+<Tabs className="language-tabs">
+  <TabItem value="as" label="🚀 - Assemblyscript">
+
+  ```ts
+    @nearbindgen
+    class Donation{
+      account_id: string,
+      amount: u128
+    }
+  ```
+
+  </TabItem>
+  <TabItem value="rs" label="🦀 - Rust">
+
+  ```rust
+    #[near_bindgen]
+    #[derive(Default, BorshDeserialize, BorshSerialize)]
+    pub struct Donation {
+      account_id: AccountId,
+      amount: Balance
+    }
+  ```
+
+  </TabItem>
+</Tabs>
+
+The `nearbindgen` decorator adds methods to the classes, so once instantiated they can be correctly serialized in the contract's state (storage). For example, the class decorated in Assemblyscript originally has only two attributes: `account_id` and `amount`. However, after being decorated it gains methods to be serialized into JSON. This is helpfull since contracts actually obtain and return values encoded in JSON!.
```

Comment:
```suggestion
The `nearbindgen` decorator adds methods to the classes, so once instantiated they can be correctly serialized in the contract's state (storage). For example, the class decorated in Assemblyscript originally has only two attributes: `account_id` and `amount`. However, after being decorated it gains methods to be serialized into JSON. This is helpful since contracts actually obtain and return values encoded in JSON!.
```